src/arkade-lightning.ts

```diff
@@ -816,6 +881,19 @@ export class ArkadeLightning {
         return inputs.every((input) => input.witnessUtxo);
     };
 
+    private verifySignatures = (
+        tx: Transaction,
+        inputIndex: number,
+        requiredSigners: string[]
+    ): boolean => {
+        try {
+            verifyTapscriptSignatures(tx, inputIndex, requiredSigners);
```

Comment:
ideally, before checking the signatures, we should verify the input witness UTXO fields because a malicious server could set wrong prevout, making the sig valid but the transaction not broadcastable onchain.

Maybe not in the scope of this PR but worth to at least add a todo comment